net/api/client/client_test.go

```diff
@@ -211,15 +199,12 @@ func TestClient_AddRecord(t *testing.T) {
 
 	// Create a thread, keeping read key and log private key on the client
 	id := thread.NewIDV1(thread.Raw, 32)
-	fk, err := symmetric.NewRandom()
-	if err != nil {
-		t.Fatal(err)
-	}
+	tk := thread.NewRandomServiceKey()
 	sk, pk, err := crypto.GenerateEd25519Key(crand.Reader)
 	if err != nil {
 		t.Fatal(err)
 	}
-	info, err := client.CreateThread(context.Background(), id, core.FollowKey(fk), core.LogKey(pk))
+	info, err := client.CreateThread(context.Background(), id, core.ThreadKey(tk), core.LogKey(pk))
```

Comment:
Here's an example of supplying a locally-created service-only key, and a locally-created private key.